src/server.js

```diff
@@ -91,20 +96,37 @@ app.use(
             createdVmContext: false,
         }: RequestStats);
 
+        /**
+         * Before we do any real work, we may have added the log middleware
+         * and we'll want everything to use it. Rather than pass that around
+         * or pass the request around, we can use our logging module and
+         * tell it to scope logging to the request logger during the request.
+         * NOTE: the $Request type doesn't have a log field, officially.
+         * $FlowIgnore
+         */
+        const requestLog: Logger = req.log;
```

Comment:
If `requestLog` can be undefined then the type probably shouldn't be `Logger`? Maybe `?Logger`?

Reply:
Oh, yeah, that seems reasonably at this point. It would be `undefined` when in dev/test. I think I wrote this line before I had determined that we wouldn't support `req.log` at all in dev/test.

Reply:
I've reworked things substantially, so this is not relevant to how the code works anymore.